config/types/schema.go

```diff
@@ -151,7 +149,7 @@ type PasswdGroup struct {
 type PasswdUser struct {
 	Create            *Usercreate        `json:"create,omitempty"`
 	Gecos             string             `json:"gecos,omitempty"`
-	Groups            []Group            `json:"groups,omitempty"`
```

Comment:
Also if it gets silly and decides to switch it back, these changes are in their own commit and could be easily reverted.